readthedocs/config/tests/test_config.py

```diff
@@ -1104,6 +1105,26 @@ def test_python_install_requirements_check_valid(self, tmpdir):
         assert isinstance(install[0], PythonInstallRequirements)
         assert install[0].requirements == 'requirements.txt'
 
+    def test_python_install_packages_check_valid(self, tmpdir):
```

Comment:
We should add tests for invalid data too, there are examples in the same file

Reply:
Done